runtime/queries/_gjs/injections.scm

```diff
@@ -0,0 +1,20 @@
+; PARSE GLIMMER TEMPLATES
+(call_expression
+  function: [
+    (identifier) @injection.language
+    (member_expression
+      property: (property_identifier) @injection.language)
+  ]
+  arguments: (template_string) @injection.content)
+
+; e.g.: <template><SomeComponent @arg={{double @value}} /></template>
+((glimmer_template) @injection.content
+ (#set! injection.language "hbs"))
+
+; Parse Ember/Glimmer/Handlebars/HTMLBars/etc. template literals
+; e.g.: await render(hbs`<SomeComponent />`)
+(call_expression
+  function: ((identifier) @_name
+             (#eq? @_name "hbs"))
+  arguments: ((template_string) @glimmer
+              (#offset! @glimmer 0 1 0 -1)))
```

Comment:
We don't support `#offset!`. This will also need to be updated to use `injection.content` and `injection.language` rather than tagging with `@glimmer` - that's the way nvim-treesitter did injections in the past.

Something like

```scm
(call_expression
  function: ((identifier) @_name
             (#eq? @_name "hbs"))
  arguments: ((template_string) @injection.content
              (#set! injection.language "glimmer")))
```

Reply:
This can probably just be dropped due to [RFC 0779](https://rfcs.emberjs.com/id/0779-first-class-component-templates)

> As of [RFC 0779][rfc-0779], we decided on `<template>` over `hbs`; see the RFC for the full rationale. ~~The `hbs` format is still technically supported by this repo for transition purposes for the early adopters who helped us get here, but will be removed at some point in the near future!~~ `hbs` has been removed -- if you rely on this feature, please use `ember-template-imports @ < v4`, until migrated to `<template>`

If anywhere this probably belongs into the javascript/typescript injections instead of gjs